{{cookiecutter.project_shortname}}/misc/header.jinja2

```diff
@@ -1,6 +1,5 @@
 {%- raw %}{#{% endraw %}
-  Copyright (C) {{ cookiecutter.year }} {{ cookiecutter.copyright_holder }}.
+{{ '  Copyright (C) %s %s.'|format(cookiecutter.year, cookiecutter.copyright_holder)|wordwrap(width=77, wrapstring='\n  ')}}
 
-  {{ cookiecutter.project_name }} is free software; you can redistribute it and/or modify it
-  under the terms of the MIT License; see LICENSE file for more details.
-{% raw %}#}{% endraw -%}
+{{ '  %s is free software; you can redistribute it and/or modify it under the terms of the MIT License; see LICENSE file for more details.'|format(cookiecutter.project_name)|wordwrap(width=77, wrapstring='\n  ')}}
+{% raw %}#}{% endraw -%} 	{% raw %}#}{% endraw -%}
```

Comment:
``{% raw %}#}{% endraw -%}`` is this one supposed to be double?

Reply:
Nope :/